sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuShuffledHashJoinExec.scala

```diff
@@ -50,21 +49,21 @@ class GpuShuffledHashJoinMeta(
 
   override def convertToGpu(): GpuExec =
     GpuShuffledHashJoinExec(
-      leftKeys.map(_.convertToGpu()),
-      rightKeys.map(_.convertToGpu()),
+      leftKeys.map(_.convertToGpu()).asInstanceOf[Seq[GpuExpression]],
```

Comment:
We recently changed things so we are used `AttributeReference` and deleted `GpuAttributeReference`.  After this change all of the APIs need to take an `Expression` and not a `GpuExpression`.  The only time that we use `GpuExpression` is after the references have been bound.

Reply:
Thanks @revans2 for taking a look and explaining why it is not required anymore. I had not understood earlier the context of `AttributeReference`. Will close this PR.